tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -9169,7 +9012,10 @@ node_modules/chrome-devtools-frontend/front_end/quick_open/FilteredListWidget.js
 node_modules/chrome-devtools-frontend/front_end/quick_open/FilteredListWidget.js(33,57): error TS2339: Property 'createChild' does not exist on type 'Element'.
 node_modules/chrome-devtools-frontend/front_end/quick_open/FilteredListWidget.js(40,50): error TS2345: Argument of type 'this' is not assignable to parameter of type '{ createElementForItem(item: T): Element; heightForItem(item: T): number; isItemSelectable(item: T): boolean; selectedItemChanged(from: T, to: T, fromElement: Element, toElement: Element): void; }'.
   Type 'FilteredListWidget' is not assignable to type '{ createElementForItem(item: T): Element; heightForItem(item: T): number; isItemSelectable(item: T): boolean; selectedItemChanged(from: T, to: T, fromElement: Element, toElement: Element): void; }'.
-    Property '_promptHistory' does not exist on type '{ createElementForItem(item: T): Element; heightForItem(item: T): number; isItemSelectable(item: T): boolean; selectedItemChanged(from: T, to: T, fromElement: Element, toElement: Element): void; }'.
+    Types of property 'createElementForItem' are incompatible.
+      Type '(item: number) => Element' is not assignable to type '(item: T) => Element'.
+        Types of parameters 'item' and 'item' are incompatible.
+          Type 'T' is not assignable to type 'number'.
```

Comment:
this exposes the bug that we don't treat class type parameters correctly in methods that are part of a prototype assignment. See #24230